app/src/androidTest/java/ch/epfl/sdp/blindly/fake_module/FakeRecordingsModule.kt

```diff
@@ -43,5 +43,11 @@ open class FakeRecordingsModule {
             callback.onSuccess()
         }
 
+        override fun deleteFile(
+            recordingPath: String,
+            callback: Recordings.RecordingOperationCallback
+        ) {
+            callback.onSuccess()
```

Comment:
Is there an onError we should take care of?

Reply:
Probably, unfortunately I haven't unlocked the skills yet to be able to test the addOnFailureListener() and callback.onFailure()
I tried to do what Tibor did when he checked for the updateEmail, but was not able to make it work :/